packages/cli/src/zosfiles/download/uss/UssFile.handler.ts

```diff
@@ -25,13 +25,20 @@ export default class UssFileHandler extends ZosFilesBaseHandler {
             stageName: TaskStage.IN_PROGRESS
         };
         commandParameters.response.progress.startBar({task});
-        return Download.ussFile(session, commandParameters.arguments.ussFileName, {
+
+        const attributes = ZosFilesAttributes.loadFromFile(
+            commandParameters.arguments.attributes,
+        );
+        const downloadOptions: IDownloadOptions = {
             binary: commandParameters.arguments.binary,
             encoding: commandParameters.arguments.encoding,
             file: commandParameters.arguments.file,
             task,
             responseTimeout: commandParameters.arguments.responseTimeout,
-            overwrite: commandParameters.arguments.overwrite
-        });
+            overwrite: commandParameters.arguments.overwrite,
+            attributes
+        };
```

Comment:
Declaration and assignment of `attributes` can be joined since it isn't used elsewhere in the function 😋 

```js
   overwrite: commandParameters.arguments.overwrite,
   attributes: ZosFilesAttributes.loadFromFile(...)
}